cmake/FindDIA2Dump.cmake

```diff
@@ -15,6 +15,6 @@ target_link_libraries(DIA2Dump PUBLIC DIASDK::DIASDK)
 target_link_libraries(
   DIA2Dump PUBLIC multicore::multicore concurrentqueue::concurrentqueue
                   oqpi::oqpi xxHash::xxHash cereal::cereal)
-target_include_directories(DIA2Dump PRIVATE "OrbitCore/")
+target_include_directories(DIA2Dump PRIVATE "OrbitCore/" "OrbitBase/include/")
```

Comment:
I do not completely understand where is this file used? can it have a dependency on OrbitBase instead?

Reply:
DIA2Dump includes headers from OrbitCore that include headers from OrbitBase, this is an atrocious hack.  Fixing it properly is quite involved, and I'm planning on switching to llvm for symbol parsing on Windows anyway, I don't want to spend any time fixing this as it will all go away soon.